shepherd/sensors/sensor_interface.py

```diff
@@ -0,0 +1,176 @@
+"""
+A process that is the interface between YDL and shared memory.
+It parses device commands from YDL and writes it to shared memory.
+It reads device data from shared memory and publishes it to YDL if necessary.
+- Ex: Provide the device uid and param index of a button
+-     This will write to YDL if the button has been pressed.
+"""
+import threading
+import time
+import queue
+# import pyximport
+# pyximport.install()
+import shm_api
+import sys
+sys.path.insert(1, '../')
+from debouncer import Debouncer
+from ydl import (
+    ydl_send,
+    ydl_start_read
+)
+from typing import List, Union
+from sensors_config import (
+    arduinos,
+    translate_ydl_message,
+    HEADER_MAPPINGS,
+    HEADER_COMMAND_MAPPINGS,
+    Parameter,
+    previous_debounced_value,
+    LowcarMessage
+)
+from utils import (
+    YDL_TARGETS,
+)
+#YDL -> TURN_ON_LIGHT, {light: 8}
```

Comment:
probably clean up these comments?

Reply:
ehh maybe its useful, but if we leave them, more explanation needs to be given

Reply:
I'm in favor of fleshing them out with more explanation, however I have no clue what they're talking about.